### DISCUSSION TITLE
Mouse mode? Open Tablet Driver

### DISCUSSION THREAD

**Discussion by Zirowx** [OpenTabletDriver/OpenTabletDriver#2356] [Q&A] [answered]
Hi, I have a Gaomon m10k pro and I am using Opentabletdriver on linux.... Is there a mouse mode? I tried looking for it but I don't see it.
On windows I used to use it this way, it was more comfortable for me.

Also I have another problem. As you can see from the picture below in pop os settings in "Wacom Tablet", my graphics tablet is recognized as Huion H950P. Is this normal or am I doing something wrong?
![Screenshot_from_2022-07-20_16-50-35](https://user-images.githubusercontent.com/109765536/180313350-7e8a3740-ebde-4c46-bb45-f12f9698bd12.png)

**Comment by jamesbt365** [comment, marked as answer]
Called relative mode

**Reply by jamesbt365** [reply]
I'll also answer your other question because why not.

Gaomon is a sub company to huion, and most tablets share 1 firmware so occurances like this can and do happen.

**Reply by Zirowx** [reply]
Ah I didn't know... I thank you.
